lib/service/TokenList.dart

```diff
@@ -111,6 +111,17 @@ class TokenList {
           idToAddress[ath[1]]![2]);
     }),
   ];
+
+  static String mapTickerToName(String ticker) {
+    return tokenList.firstWhere((token) => token.ticker == ticker).name;
```

Comment:
If ticker is not present in the token list you'll get a `Bad state: no element`. Are you sure ticker is always gonna be in the list? if not you should use `firstWhereOrNull` and handle the missing presence.